components/ScoreTable/index.tsx

```diff
@@ -1,10 +1,10 @@
-import React, { useMemo } from 'react';
-import { useTable, useSortBy, usePagination } from 'react-table';
+import React, { useMemo, useState } from 'react';
+import { useTable, useSortBy, usePagination, useFilters } from 'react-table';
 import styled from 'styled-components';
 // import Skeleton, { SkeletonTheme } from 'react-loading-skeleton';
 // import Link from '../../components/LinkWithSeason';
-import { Player, Goalie } from '../..';
-
+import { Player, Goalie, SearchType } from '../..';
+import SearchBar from '../SearchBar/SearchBar'
```

Comment:
using a search type interface to specify objects with human readable text and the id for the react-table